data/opam_docs/2.1/03_faq.md

```diff
@@ -0,0 +1,446 @@
+field. Opam should print the required system dependencies, as documented for
+your OS, upon failure, and the `depext` opam plugin can take care of installing
+them for you:
+
+```
+opam depext <opam-packages>
+```
+
+This should install `opam-depext` if needed and prompt to install the system
+packages required by your opam packages or their dependencies, through your OS's
+packaging system.
+
+If that doesn't work...
+* Check for hints printed by the failing package
+* Lookup the development packages corresponding to the error in your system's
+  package repositories.
+* Dependencies for your specific system may not be known, but check the output
+  of `opam list --rec --resolve <package>,<package>... --columns name,depexts:`:
+  it will list dependencies on all known systems and may get you in the right
+  direction.
+
+In any of these cases, that's useful information that was missing from the opam
+repository: we would really appreciate that you take a minute to save others the
+trouble of looking by filling an issue in
+[the opam-repository tracker](https://github.com/ocaml/opam-repository/issues),
+with your system details, the output of `opam config report`, and the solution,
+if you found it. Thanks!
+
+---
+
+#### 🐫  I have weird checksum errors: where do they come from?
+
+First of all, you should make sure your repositories are up-to-date:
+
+```
+opam update
+```
+
+If this isn't enough, or if you get the checksum errors while running `opam
+init`, this could be caused by a badly configured proxy cache that is serving
+stale files. To clear your proxy cache, you can use `wget --no-cache
+<remote-file>` and retry.
+
+As a last resort, you can bypass the checksum checks using `--no-checksums`.
+
+---
+
+#### 🐫  opam is prompting me to install or upgrade packages that I am not interested in, or doesn't install the latest version by default. Why? What can I do?
+
+* You can be more explicit in your request (`opam upgrade PACKAGES`, `opam
+  install 'PACKAGE>=VERSION' PACKAGE...`, etc.). The latest version may not be
+  available on your system, in this case this will tell you why.
+* See how to set [solver preferences](External_solvers.html) that could match your intentions better than the defaults
+* Check for pending reinstallations `opam reinstall --list-pending`
+
+---
+
+#### 🐫  When trying to install a new package, opam wants to remove or downgrade packages that I have installed. How to know why?
+
+There is likely a conflict between them or their dependencies and what you are
+requesting, here is how to find out. We'll suppose you were trying to install
+`foo` and `bar` got removed:
+
+* `opam install foo bar`, if not possible, will tell you why.
+* The above may find a solution by using older version of the packages, in that
+  case try and force the latest versions thusly: `opam install foo.2.0 bar.1.1`
+  (you can also use constraints like `'foo>=2.0'`).
+
+---
+
+#### 🐫  Where do I report Bugs, Issues and Feature Requests?
```

Comment:
```suggestion
#### 🐫  Where do I report bugs, issues, and feature requests?
```
While you know I prefer title case for headers and subheaders, since all these subheaders have more than 7 words and are phrased as sentences or questions, it's best to use sentence case.